pkg/controller/populators/populator-base.go

```diff
@@ -223,7 +228,7 @@ var desiredAnnotations = []string{cc.AnnPodPhase, cc.AnnPodReady, cc.AnnPodResta
 	cc.AnnPreallocationRequested, cc.AnnPreallocationApplied, cc.AnnCurrentCheckpoint, cc.AnnMultiStageImportDone,
 	cc.AnnRunningCondition, cc.AnnRunningConditionMessage, cc.AnnRunningConditionReason}
 
-func (r *ReconcilerBase) updatePVCWithPVCPrimeAnnotations(pvc, pvcPrime *corev1.PersistentVolumeClaim, updateFunc updatePVCAnnotationsFunc) error {
+func (r *ReconcilerBase) updatePVCWithPVCPrimeAnnotations(pvc, pvcPrime *corev1.PersistentVolumeClaim, updateFunc updatePVCAnnotationsFunc) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
kind of weird to have different return values between `updatePVCWithPVCPrimeAnnotations` and `updatePVCWithPVCPrimeLabels` no?

Reply:
In an earlier rev of the PR they looked the same, but @ksimon1 suggested making it simpler. Leaving it up to you to decide. https://github.com/kubevirt/containerized-data-importer/pull/3103#discussion_r1524503076

Reply:
it's fine can keep